LICENSE

```diff
@@ -1,6 +1,7 @@
 The MIT License (MIT)
 
 Copyright (c) 2015 zserge
+Copyright (c) 2019 Anvil contributors
```

Comment:
Inkrement contributors? :new_moon_with_face:

Reply:
Done. I'll leave everything else for future MR.

Reply:
Done. I'll leave everything else for the future MR.